src/api/app/controllers/webui/cloud/azure/configurations_controller.rb

```diff
@@ -10,6 +10,8 @@ class ConfigurationsController < WebuiController
 
         # GET /cloud/azure/configuration
         def show
+          # FIXME: remove it when cloud upload are ready to production
+          switch_to_webui2 if Rails.env.development?
```

Comment:
For this action, the view is already fully migrated in this PR. We don't need to check for the environment.

Reply:
Check removed. Comment removed too.